docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -37,47 +37,61 @@ Option                                 Usage
 Output
 ^^^^^^
 
-Values are given as *charge density* per phase space bin.
-In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume:
+The histograms are stored in ``.hdf5`` files in the ``simOutput/...`` directory.
 
-.. code:: python
+...
 
-   ps = ... # get data set from h5py
+Values are given as *charge density* per phase space bin.
+In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume ``dV``.
 
-   # 3D3V example
-   dV = ps.attrs['dV'] * ps.attrs['dr_unit']**3
-   charge_per_bin = ps[:,:] * ps.attrs['sim_unit'] * dV
+Analysis
+^^^^^^^^
 
-Out-of-Range Behavior
-^^^^^^^^^^^^^^^^^^^^^
+The easiest way is to load the data in Python:
 
-Particles that are *not* in the range of ``<ValL>``/``<ValR>`` get automatically mapped to the lowest/highest bin respectively.
-Take care about that when setting your range and during analysis of the results.
+.. code:: python
 
-Spatial Offset
-^^^^^^^^^^^^^^
+   from picongpu.plugins.phase_space import PhaseSpace
+   import matplotlib.pyplot as plt
+   from matplotlib.colors import LogNorm
+   import numpy as np
 
-The spatial extent of the output might change due to a moving window.
-Additional attributes are provided to retrieve that spatial information:
 
-See this python example:
+   # load data
+   phase_space = PhaseSpace('/home/axel/runs/foil_001')
+   e_ps, e_ps_meta = phase_space.get('e', ps='ypy', iteration=1000)
 
-.. code:: python
+   # unit conversion from SI
+   mu = 1.e6  # meters to microns
+   e_mc_r = 3.6618e21  # electrons: kg * m / s to beta * gamma
```

Comment:
OPTIONAL:
I would prefere loadung scipy constanst and then:
```python
import scipy.constants as const
e_mc_r = 1./(const.electron_mass * const.speed_of_light) # electrons: kg * m / s to beta * gamma
```

Reply:
that's true, but also pulls in a rather huge dependency

Reply:
I know - iy you do not want to add a scipy depency how about 
```python
e_mc_r = 1./(9.1e-31 * 2.9979e8)
```
instead: at least these magic numbers are more familiar for physicists

Reply:
that's true. I am also thinking about adding `m_species * c` to the output of PS just to make it easier for integrated analysis not to have a table of each species' mass by hand.